org.gecko.emf.exporter.ods/bnd.bnd

```diff
@@ -13,3 +13,7 @@ Bundle-Description: ODS (OpenDocument Spreadsheet) Exporter for EMF
 	slf4j.api
 
 Private-Package: org.gecko.emf.exporter.ods.impl
+
+Import-Package: \
```

Comment:
`Import-Package` can be removed, as the Bundle you add manually only appear to contain Constants.

Reply:
The Private-Package can be removed as well, as the bnd default will package everything correctly.